gradle.properties

```diff
@@ -1,3 +1,3 @@
 org.gradle.daemon=true
-org.gradle.jvmargs=-Xms256m -Xmx1024m
```

Comment:
This may cause builds to crash on some devices. Builds work fine for me, so I would recommend you only set this for your computer.